client/me/purchases/controller.jsx

```diff
@@ -51,10 +51,11 @@ function renderPage( context, component ) {
 	);
 }
 
-function setTitle( ...title ) {
-	titleActions.setTitle(
+// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+function setTitle( context, ...title ) {
```

Comment:
You missed `addCardDetails` in this file for passing `context` as the first argument.